app/modules/ads/bill-the-lizard-wrapper.js

```diff
@@ -99,6 +99,8 @@ export const billTheLizardWrapper = {
     let baseSlotName = 'incontent_boxad_1';
     defaultStatus = NOT_USED_STATUS;
 
+    context.set('options.cheshireCatFirstSlot', 'top_boxad');
```

Comment:
Technicaly we still can revert `top_boxad` switch changes. Let's move it into `if` clause below with:
`context.set('options.cheshireCatFirstSlot', baseSlotName);`

Reply:
done